src/core/geometry/qgsabstractgeometry.h

```diff
@@ -88,7 +88,13 @@ class CORE_EXPORT QgsAbstractGeometry
     //! Segmentation tolerance as maximum angle or maximum difference between approximation and circle
     enum SegmentationToleranceType
     {
+
+      /** Maximum angle between generating radii (lines from arc center
+       * to output vertices) */
       MaximumAngle = 0,
```

Comment:
It's a bit ugly for longer strings like this, but see if changing to

    MaximumAngle = 0, //!< Maximum angle between generating radii (lines from arc center to output vertices)

works for you